flink-runtime-web/web-dashboard/.stylelintrc.js

```diff
@@ -0,0 +1,284 @@
+module.exports = {
+  plugins: ['stylelint-order'],
+  extends: ['stylelint-config-standard', 'stylelint-prettier/recommended'],
+  rules: {
+    'prettier/prettier': [
+      true,
+      {
+        singleQuote: false
+      }
+    ],
+    'no-empty-source': null,
+    'no-descending-specificity': null,
+    'no-invalid-position-at-import-rule': null,
+    'function-calc-no-invalid': null,
+    'font-family-no-missing-generic-family-keyword': null,
+    'selector-pseudo-element-no-unknown': [
+      true,
+      {
+        ignorePseudoElements: ['ng-deep']
+      }
+    ],
+    // Reference: https://github.com/hudochenkov/stylelint-config-hudochenkov/blob/master/order.js
+    'order/order': [
+      [
+        'dollar-variables',
+        'custom-properties',
+        'at-rules',
+        'declarations',
+        {
+          type: 'at-rule',
+          name: 'supports'
+        },
+        {
+          type: 'at-rule',
+          name: 'media'
+        },
+        'rules'
+      ],
+      {
+        severity: 'warning'
+      }
+    ],
+    'order/properties-order': [
```

Comment:
Well, this piece of info is taken from: https://github.com/hudochenkov/stylelint-config-hudochenkov/blob/master/order.js. The author of this config is actually a member of Stylelint team. There is a comment about this at line 22.

Reply:
After thinking over this, maybe it is not a good idea to directly use this config's source code. I think I am going to add it as a dev dependency and extends this config in the `.stylelintrc.js` file as the README suggests.

Reply:
After thinking over it, it may not be a good idea to extract the source code from this config. Instead, I add it as a dev dependency and extend it in the `.stylelintrc.js` file as the README suggests.